README.md

```diff
@@ -22,6 +22,23 @@ npm i
 npm run install-extension
 ```
 
+## Test your changes on your local install of VS Code
+
+Firstly, uninstall the official release of the extension and reload VS Code if you have it installed. You will need to do this step each time you want to re-install and test your local version.
+
+Install `vsce`:
+
+```sh
+npm install -g vsce
```

Comment:
1. This won't work without `sudo`.
2. I'm generally against installing _anything_ npm globally, due to the general brittleness of npm.
3. Running `npm i` from the previous section will install `vsce` at the correct version. This is important, since just installing the latest version isn't guaranteed to be compatible.

Suggest changing this to just use `npm i` or `npm ci`.